bindings/nodejs/lib/types/block/output/irc-27.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { utf8ToHex } from "../../../utils";
+
+/**
+ * The IRC27 NFT standard schema.
+ */
+class Irc27Metadata {
+    /** The IRC standard */
+    readonly standard: string = "IRC27";
+    /** The current version. */
+    readonly version: string = "v1.0";
+    /** The media type (MIME) of the asset.
+     * 
+     * ## Examples
+     * - Image files: `image/jpeg`, `image/png`, `image/gif`, etc.
+     * - Video files: `video/x-msvideo` (avi), `video/mp4`, `video/mpeg`, etc.
+     * - Audio files: `audio/mpeg`, `audio/wav`, etc.
+     * - 3D Assets: `model/obj`, `model/u3d`, etc.
+     * - Documents: `application/pdf`, `text/plain`, etc.
+     */
+    type: string;
+    /** URL pointing to the NFT file location. */
+    uri: string;
+    /** The human-readable name of the native token. */
+    name: string;
+    /** The human-readable collection name of the native token. */
+    collectionName?: string;
+    /** Royalty payment addresses mapped to the payout percentage. */
+    royalties: Map<string, number> = new Map();
+    /** The human-readable name of the native token creator. */
+    issuerName?: string;
+    /** The human-readable description of the token. */
+    description?: string;
+    /** Additional attributes which follow [OpenSea Metadata standards](https://docs.opensea.io/docs/metadata-standards). */
+    attributes: Array<Attribute> = new Array();
```

Comment:
`The array literal notation [] is preferable`
I think that would be
```suggestion
    attributes: Attribute[] = new Array();
```

Reply:
should the default just be `[]` then?

Reply:
Don't know, `The array literal notation [] is preferable` was generated by the CI

Reply:
should be able to just set "attributes: Attribute[];" dont think the new is needed

Reply:
I don't want it to be undefined tho